content/definitions/czym-sa-dumb-smart-komponenty.md

```diff
@@ -0,0 +1,34 @@
+---
+question: Czym są dumb i smart komponenty?
+---
+
+`Smart komponenty` wykonują logiczne operacje, strzelają po dane, wykonują obliczenia, modyfikują dane, które później mogą
+przekazać do innych komponentów. Często `smart komponenty` nazywamy kontenerami, których zadaniem jest właśnie przejęcie całego mechanizmu strzelania po API, czy
+operacji na danych.
+

```

Comment:
I would mention that this approach is mostly used in JS frameworks like React or Angular.